functional_tests/bin/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
The normandy container can't use `wait-for-it.sh` on itself, as far as I can tell. So I copied it to the `functionaltestrunner` container. I'd like to not have multiple copies lying around but don't have a good alternative.